src/pages/issues/index.tsx

```diff
@@ -10,24 +10,43 @@ import {
 import { ISSUES_URL } from '@/constants/url';
 import { IssueItem } from '@/interfaces/issueItem.type';
 import { PullRequestAndIssueItem } from '@/interfaces/pullRequestIssueItem';
+import { useRouter } from 'next/router';
+import { getQueryString } from '@/utils/getQueryString';
 
 type SearchFieldProps = {
     onSearchTextSubmitted: (searchString: string) => void;
     loading: boolean;
 };
 
 const SearchField = ({ onSearchTextSubmitted, loading }: SearchFieldProps) => {
+    const router = useRouter();
+    const dev = router?.query?.dev;
     const [searchText, setSearchText] = useState<string>('');
     const onSearchTextChanged = (e: ChangeEvent<HTMLInputElement>) => {
         setSearchText(e.target.value);
     };
 
+    const queryParamValue = getQueryString(searchText);
+    const updateQueryString = () => {
+        router.push({
+            query: {
+                ...router.query,
+                q: `search:${queryParamValue.text.toLowerCase()}`,
```

Comment:
Can we do this lowercase transformation inside `getQueryString()` util function

Reply:
done

Reply:
Unnecessary `toLowerCase` needs to be removed from here.